hypervisor/virt/esx/README.md

```diff
@@ -0,0 +1,84 @@
+# Hypervisor-builder: ESX hypervisor
```

Comment:
I think here we can also use ```Hypervisor-builder: vCenter```, don't need to match the ```ESX``` mode name in virt-who.

Reply:
OK